lib/Remotery.c

```diff
@@ -5205,51 +5206,32 @@ RMT_API void _rmt_LogText(rmtPStr text)
     start_offset = 8;
 
     // There might be newlines in the buffer, so split them into multiple network calls
-    prev_offset = start_offset;
+    offset = start_offset;
     for (i = 0; text[i] != 0; i++)
     {
         char c = text[i];
 
         // Line wrap when too long or newline encountered
-        if (prev_offset == sizeof(line_buffer) - 3 || c == '\n')
+        if (offset == sizeof(line_buffer) || c == '\n')
         {
-            if (QueueLine(g_Remotery->mq_to_rmt_thread, line_buffer, prev_offset, ts) == RMT_FALSE)
+            if (QueueLine(g_Remotery->mq_to_rmt_thread, line_buffer, offset, ts) == RMT_FALSE)
                 return;
 
             // Restart line
-            prev_offset = start_offset;
-        }
-
-        // Safe to insert 2 characters here as previous check would split lines if not enough space left
-        switch (c)
-        {
-            // Skip newline, dealt with above
-            case '\n':
-                break;
-
-            // Escape these
-            case '\\':
-                line_buffer[prev_offset++] = '\\';
-                line_buffer[prev_offset++] = '\\';
-                break;
-
-            case '\"':
-                line_buffer[prev_offset++] = '\\';
-                line_buffer[prev_offset++] = '\"';
-                break;
+            offset = start_offset;
 
-            // Add the rest
-            default:
-                line_buffer[prev_offset++] = c;
-                break;
+            if (c == '\n')
```

Comment:
Can you leave a comment about what this is doing? The more readable code would be to check `c != '\n'` before adding to `line_buffer` below but this adds an extra branch per character.

Reply:
Right, I wanted to avoid the extra branch per character that was not needed anymore after removing the \\ and \" escaping.
Added a comment now, though.